mips/intr.c

```diff
@@ -138,6 +138,7 @@ void syscall_handler(exc_frame_t *frame) {
 }
 
 void fpe_handler(exc_frame_t *frame) {
+  (void)frame;
```

Comment:
I'm not quite sure, but in case of unused parameters it should be enough to change function header as such:

```c
 void fpe_handler(exc_frame_t *) {
```